examples/json-wrapper-ext/uniffi.toml

```diff
@@ -0,0 +1,8 @@
+[bindings.python]
```

Comment:
(unrelated to this PR, but there's something smelly about the fact noting works until our examples all add this boilerplate)